wordpress-mobile/WordPress-iOS
# New Reader Navigation - Tab Bar and Buttons

Ref #13322

#### This PR Replaces the existing reader with the new Reader Navigation, ad described in the design: https://wp.me/pauD4L-Dx

##### Notes

- This PR only adds the tab bar and buttons UI. Reader content and actions will be added in future PRs
- To avoid blocking the existing reader, the feature flag will be set to false before merging into develop
- It is expected that some ui tests might break at this point; again, the feature will not be active in 'develop' (feature flag set to false) until the new reader is not fully functional.

To test:
- build/run from Xcode or download the installable build
- go to the Reader
- verify that the tab bar and buttons match the design specs linked above
- Verify that the buttons bar only appears in the 'Following' tab
- To test the 'reset filter' button, tap on the 'Filter'
  - Verify that the title changes and the 'reset filter button' ( x ) is displayed
  - Tap the reset filter button, and verify that the 'filter button' returns to its original state ('Filter') and that the 'reset filter button' does not show

PR submission checklist:
